src/routes/api.py

```diff
@@ -1,12 +1,20 @@
-from fastapi import FastAPI, HTTPException
-from utils.logging import get_logger
+from fastapi import FastAPI
+
+from src.utils.logging import get_logger
+from src.services.chat import init_chat, process_chat_request
 
 logger = get_logger(__name__)
 
 app = FastAPI()
 
 
+@app.get("/chat/init")
```

Comment:
There is a [official blog](https://fastapi.tiangolo.com/tutorial/bigger-applications/) from FastAPI about defining routes if they are in separate files. Shall we follow that guidelines?

Reply:
I will consider this in the next PR.